avm/utilities/e2e-template-assets/templates/azure-stack-hci/modules/azureStackHCIHost/README.md

```diff
@@ -0,0 +1,67 @@
+# HCI Azure Host Deployment Bicep Module
```

Comment:
How come you have this readme here?  Is this content you wanted to have in the  module's README.md? 
If so (in case you did not know), you can add a `## Notes` section to it (and as many subheaders below it as you want) and add the content there. The readme generation will preserve that section and append it after the generation again to the file

Reply:
This is more because this code is reusable elsewhere (in labs, etc) and I wanted to document it for easier sharing. I could remove the readme of you prefer...